### ISSUE TITLE
one-way "cross-posted to" indicator

### ISSUE THREAD

**Issue by Die4Ever** [LemmyNet/lemmy-ui#2528]
### Requirements

- [X] This is a bug report, and if not, please post to https://lemmy.ml/c/lemmy_support instead.
- [X] Please [check](https://github.com/LemmyNet/lemmy-ui/issues) to see if this issue already exists.
- [X] It's a single bug. Do not report multiple bugs in one issue.
- [X] It's a frontend issue, not a backend issue; Otherwise please create an issue on the [backend repo](https://github.com/LemmyNet/lemmy) instead.

### Summary

I look at this post and it says "cross-posted to:apple_enthusiast@lemmy.world`

https://programming.dev/post/15341921

![image](https://github.com/LemmyNet/lemmy-ui/assets/30947252/a62a86fd-bbcd-43de-b7f5-0229f93b074b)

I go to the cross post and it does not detect the other side of the cross post

https://programming.dev/post/15298393

![image](https://github.com/LemmyNet/lemmy-ui/assets/30947252/f58c1a4b-2c35-4ce4-9772-c7af4de8f6ff)

### Steps to Reproduce

1. Check link https://programming.dev/post/15341921
2. Compare with link https://programming.dev/post/15298393

### Technical Details

Windows 10, Chrome.

### Lemmy Instance Version

0.19.3

### Lemmy Instance URL

https://programming.dev/post/15341921

**Comment by Die4Ever** [comment]
I think bot posts are not showing up, the screenshots in the OP show that, and I saw another example today...

https://programming.dev/post/15386164

![image](https://github.com/LemmyNet/lemmy-ui/assets/30947252/93b384e3-e05f-4a1f-81b2-ab5ca92c88ef)

https://programming.dev/post/15405661

![image](https://github.com/LemmyNet/lemmy-ui/assets/30947252/87faeada-d1c6-4a54-b3da-dcb92e7b38b4)

**Comment by dessalines** [comment]
Go to your user settings, do you have `hide_bot_posts` enabled?

**Comment by Die4Ever** [comment]
> Go to your user settings, do you have `hide_bot_posts` enabled?

No, I was the one who saw the bot post on my feed and decided to cross post it

**Comment by dessalines** [comment]
I think I found the issue, its a back-end one having to do with cross_posts.
